crutest/src/main.rs

```diff
@@ -844,7 +847,7 @@ async fn main() -> Result<()> {
         pr = None;
     }
 
-    let _join_handle = up_main(crucible_opts, opt.gen, None, io, pr)?;
+    let _join_handle = up_main(crucible_opts.clone(), opt.gen, None, io, pr)?;
```

Comment:
This seems to build fine without the added `.clone()`

Reply:
Ah, that was leftover from when I did need it for code that I've since removed.
I'm suprosed clippy did not find that.

Reply:
Fixed.